static/css/common.css

```diff
@@ -1,3 +1,7 @@
-.hidden {
+div {
+  display: flex;
```

Comment:
Indeed this is global. I don't think there's any scenario when we don't want divs to display as flex boxes. Developer intent is probably never "make this a block where children are displayed following dubious layout rules that nobody remembers". Pretty much every time I see a div I end up making it display as flex. But I agree that this might be subjective so I can change this (and end up making all our divs explicitly flexbox anyway).